src/themes/teams/components/Icon/iconStyles.ts

```diff
@@ -19,6 +19,8 @@ const sizes = new Map([
   ['massive', 64],
 ])
 
+const modifiedSizes = new Map([['x-large', 24], ['xx-large', 28]])
```

Comment:
this one is used in `getSize` method only - lets inline this mapping there (actually, there is no particular reason to use `Map` type for that). This will keep things cohesive